server/src/main/java/org/apache/gravitino/server/web/rest/PermissionOperations.java

```diff
@@ -173,8 +184,10 @@ public Response revokeRolesFromGroup(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "grant-privilege-to-role." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "grant-privilege-to-role", absolute = true)
+  @AuthorizationExpression(expression = "METALAKE::OWNER || METALAKE::MANAGE_GRANTS")
   public Response grantPrivilegeToRole(
```

Comment:
If the user is the owner of the metadata object, he can grant the privilege to any role.